PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -224,33 +156,15 @@ def nanoAOD_activateVID(process):
     for modname in electron_id_modules_WorkingPoints_nanoAOD.modules:
         setupAllVIDIdsInModule(process,modname,setupVIDElectronSelection)
 
-    electronTask_ = process.egmGsfElectronIDTask.copy()
-    electronTask_.add(electronTask.copy())
-    process.electronTask = electronTask_.copy()
-    for modifier in run2_miniAOD_80XLegacy,run2_nanoAOD_94XMiniAODv1,run2_nanoAOD_94XMiniAODv2,run2_nanoAOD_94X2016,run2_nanoAOD_102Xv1,run2_nanoAOD_106Xv1:
-        modifier.toModify(process.electronMVAValueMapProducer, src = "slimmedElectronsUpdated")
-        modifier.toModify(process.egmGsfElectronIDs, physicsObjectSrc = "slimmedElectronsUpdated")
+    process.electronTask.add( process.egmGsfElectronIDTask )
```

Comment:
For my education, when is a copy of the task needed and when not?

Reply:
beats me :-)